cli/cli_app.go

```diff
@@ -61,6 +62,14 @@ func CreateTerragruntCli(version string) *cli.App {
    Moreover, for the apply and destroy commands, Terragrunt will first try to acquire a lock using DynamoDB. For
    documentation, see https://github.com/gruntwork-io/terragrunt/.`
 
+	app.Flags = []cli.Flag{
```

Comment:
I also saw logs like `[terragrunt] 2016/10/14 11:45:07 Running command:` without the `--config` flag